src/classad/lexer.cpp

```diff
@@ -456,20 +456,6 @@ tokenizeNumber (void)
 			CLASSAD_EXCEPT("Should not reach here");
 	}
 
-	if ( jsonLex ) {
-		f = Value::NO_FACTOR;
-	} else {
-		switch( toupper( ch ) ) {
-		case 'B': f = Value::B_FACTOR; wind( ); break;	
-		case 'K': f = Value::K_FACTOR; wind( ); break;
-		case 'M': f = Value::M_FACTOR; wind( ); break;
-		case 'G': f = Value::G_FACTOR; wind( ); break;
-		case 'T': f = Value::T_FACTOR; wind( ); break;
-		default:
-			f = Value::NO_FACTOR;
-		}
-	}
-
```

Comment:
If we're going to leave variable NumberFactor f here, we should initialize it to Value::NO_FACTOR.

Reply:
FWIW, my thought is to leave the `NumberFactor` thoughout the code for a release or two, just to make sure we don't need to undo this change in a hurry, then rip out all remnants of this feature.

Reply:
I agree with that idea. My main point is that we shouldn't pass around an initialized variable.